### DISCUSSION TITLE
The new license can use for pyarmor old version?

### DISCUSSION THREAD

**Discussion by pengdake** [dashingsoft/pyarmor#1185] [Q&A] [answered]
I'm just try in pyarmor 6.8.1. If I buy the license, it can be used for pyarmor 6.8.1?

**Comment by jondy** [comment, marked as answer]
Pyarmor 8 license can't be used for pyarmor < 8.
If prefer to old pyarmor, old Pyarmor License could work with pyarmor 6.8.1

**Reply by pengdake** [reply]
Thanks for your reply. But how to get old pyarmor license for pyarmor 6.8.1? Use this url(https://order.shareit.com/cart/add?vendorid=200089125&PRODUCT[300871197]=1)?

**Reply by jondy** [reply]
Yes, it's old license link.

> Thanks for your reply. But how to get old pyarmor license for pyarmor 6.8.1? Use this url([https://order.shareit.com/cart/add?vendorid=200089125&PRODUCT[300871197]=1](https://order.shareit.com/cart/add?vendorid=200089125&PRODUCT%5B300871197%5D=1))?
